### PR TITLE
Update tensorflow.md

### PR DESCRIPTION
**Pull request by knisar**
Description
-----------
In newer Keras/TF, the method to clear a metric has changed.

📄 **[View preview links for changed pages](https://github.com/wandb/docs/pull/1640#issuecomment-3286794641)**

### CODE REVIEW

**Comment by knisar** [comment]
PR to update the colab notebook: https://github.com/wandb/examples/pull/610

**Review by mdlinville — APPROVED** [review]
